module/src/main/scala/com/gu/googleauth/actions.scala

```diff
@@ -117,8 +148,8 @@ trait Actions extends UserIdentifier {
     * Looks up user's Google Groups and ensures they belong to any that are required. Redirects to
     * `failureRedirectTarget` if the user is not a member of any required group.
     */
-  def enforceGoogleGroups(userIdentity: UserIdentity, requiredGoogleGroups: Set[String], googleGroupChecker: GoogleGroupChecker, errorMessage: String = "Login failure. You do not belong to the required Google groups")
-                         (implicit request: RequestHeader): EitherT[Future, Result, Unit] = {
+  private def enforceGoogleGroups(userIdentity: UserIdentity, requiredGoogleGroups: Set[String], googleGroupChecker: GoogleGroupChecker, errorMessage: String = "Login failure. You do not belong to the required Google groups")
```

Comment:
Yes, good spot. This one I did a search and found it wasn't used by anything of ours, but perhaps that was too hasty.

In terms of relaxing access, I was doing this for RiffRaff and this allows me to avoid repeating logic in RiffRaff, as currently is the case because of the additional requirements it imposes on who can login, that aren't tied to a group.